process/interface.go

```diff
@@ -674,6 +674,7 @@ type BuiltinFunction interface {
 		acntSnd, acntDst state.UserAccountHandler,
 		vmInput *vmcommon.ContractCallInput,
 	) (*big.Int, error)
+	GasUsed() uint64
```

Comment:
GasCost?

Reply:
GasUsed is ok, as it returns how much gas was used.